src/package1/package1Version.ts

```diff
@@ -125,8 +125,14 @@ export class Package1Version implements IPackageVersion1GP {
     const query = `SELECT Id,MetadataPackageId,Name,ReleaseState,MajorVersion,MinorVersion,PatchVersion,BuildNumber FROM MetadataPackageVersion ${
       id ? `WHERE MetadataPackageId = '${id}'` : ''
     } ORDER BY MetadataPackageId, MajorVersion, MinorVersion, PatchVersion, BuildNumber`;
+    const maxFetch = env.getNumber('SF_ORG_MAX_QUERY_LIMIT') ?? 10_000;
```

Comment:
it's not necessary to pass in the maxFetch since sfdx-core do exactly what line 128 is doing automatically https://github.com/forcedotcom/sfdx-core/blob/e7c496ef7b004037b41862c46173de9d27608ad5/src/org/connection.ts#L370-L371

I think what we were really missing was just `autoFetch:true`